plugins/linear-genome-view/src/LinearGenomeView/components/SearchBox.tsx

```diff
@@ -51,11 +51,13 @@ function SearchBox({
   // 3) else assume it's a locstring and navigate to it
   async function handleSelectedRegion(option: BaseResult) {
     try {
+      // @ts-ignore
```

Comment:
probably don't need ts-ignore's?

Reply:
I was getting an error here earlier but it seems gone -- probably a fluke. Removed.